modules/test/playwright/tests/commerce/commerce-product-definitions-web/commerceAdminProductConfigurations.spec.ts

```diff
@@ -923,29 +924,7 @@ test('LPD-37886 Can filter configuration entries dataset', async ({
 		true
 	);
 
-	try {
-		await expect(
-			(
-				await commerceAdminProductConfigurationEntriesPage.tableRow(
-					0,
-					product1.name['en_US'],
-					true
-				)
-			).row
-		).toHaveCount(0);
-		await expect(
-			(
-				await commerceAdminProductConfigurationEntriesPage.tableRow(
-					0,
-					product2.name['en_US'],
-					true
-				)
-			).row
-		).toHaveCount(0);
-	}
-	catch (error) {
-		expect(error).toBeDefined();
-	}
+	await expect(page.getByText('No Results Found')).toBeVisible();
```

Comment:
can you please creator a locator for this in `commerceAdminProductConfigurationEntriesPage` ?